src/entry-point-client/PageList.tsx

```diff
@@ -13,7 +13,7 @@ export interface PageInfo {
 }
 
 function getPageTitle(...parts: string[]) {
-  return [...parts, "COVID-19 Dashboard"].join(" • ");
+  return ["Recidiviz", "Covid-19 Incarceration Model", ...parts].join(" • ");
```

Comment:
I'm not a copyeditor, so take this with a grain of salt, but I think COVID-19 should be capitalized. The official acronym of the disease is in all-caps.

Reply:
I had been doing the same, @szhu, but the mocks from Andrew officially call for Covid now. That was part of the motivation behind this PR.

Reply:
Thanks @szhu . Yes, following Andrew's guidance here.